app/mailers/user_mailer.rb

```diff
@@ -220,7 +220,7 @@ def challenge_assignment_notification(collection_id, assigned_user_id, assignmen
     @request = (assignment.request_signup || assignment.pinch_request_signup)
     mail(
       to: @assigned_user.email,
-      subject: "[#{ArchiveConfig.APP_SHORT_NAME}][#{@collection.title}] Your Assignment!"
+      subject: t("user_mailer.challenge_assignment_notification.subject", app_name: ArchiveConfig.APP_SHORT_NAME, collection_title: @collection.title).to_s
```

Comment:
I don't think we usually need `to_s` on the end -- did this not work without it?